index.html

```diff
@@ -0,0 +1,208 @@
+                </p>
+
+                <div class="date">
+                    2015.10.15(THU) ~ 16(FRI)
+                </div>
+
+                <div class="place">
+                    @ In some place in México to be defined soon
+                </div>
+            </div>
+        </div>
+        
+    
+        <div class="main-program">
+            <h2 class="subtitle">Main program</h2>
+            <div class="line"></div>
+            <div class="wrapper-items">
+                <div class="program-item">
+                    <img src="./images/ponents_icon.png" class="icon-main-program">
+                    <div class="activity">Lecture</div>
+                    <div class="activity-info">Listen to speakers from various countries about the recent advancements in electromagnetic theory.</div>
+                </div>
+                <div class="program-item">
+                    <img src="./images/exhibition_icon.png" class="icon-main-program">
+                    <div class="activity">Exhibition</div>
+                    <div class="activity-info">Appreciate new and old theories applying the concept of Faraday's tube by renowed scientists
+                    </div>
+                </div>
+        
+                <div class="program-item">
+                    <img src="./images/forum_icon.png" class="icon-main-program">
+                    <div class="activity">Forum</div>
+                    <div class="activity-info">Have the time to share your thoughts and opinions with experts for each topic.</div>
+                </div>
+        
+                <div class="program-item">
+                    <img src="./images/workshop_icon.png" class="icon-main-program">
+                    <div class="activity">Workshop</div>
+                    <div class="activity-info">Try creating your own work using your own eperiments rather than just watching at it.</div>
+                </div>
+                
+                <div class="program-item">
+                    <img src="./images/ignite.png" class="icon-main-program">
+                    <div class="activity">Ignite</div>
+                    <div class="activity-info">Get opportunities to network with CC affiliates around the world, also after the summit.</div>
+                </div>                
+                
+            </div>
+            
+            
+            <button class="orange-button">
+                Join CC Global Summit 2015
+            </button>
+
+            <div> <a href="#"> See the whole program </a></div>
+            
+    
+        </div>
+            
+        
+        <div class="title-speakers">
+            <h2 class="text-style"> Featured Speakers</h2>
+            <div class="line"></div>
+        </div>
+                    
+
+        <div class="speakers"> </div>
+        <div class="speakers-bis"> </div>
```

Comment:
- Kindly your speaker list has a large gap and lists only two for the desktop version, please minimize the gap and list 6 as the given design 👍👍



| Your design                                                             | Figma design                                         |
| --------------------------------------------------------------------------------| --------------------------------------------------------- |
|<img src="https://user-images.githubusercontent.com/52010789/210961132-38c6d9f3-49b4-4b89-a668-6e6a796fe4b3.png" width="200">|<img src="https://user-images.githubusercontent.com/52010789/210961115-86d6a704-2b09-4945-83ba-e6e27ae63b8e.png" width="200">|


😎️

🤝️